advanced_source/dispatcher.rst

```diff
@@ -32,13 +32,16 @@ Defining schema and backend implementations
 
 The general principle behind the dispatcher is that it divides the
 implementation of an operator into multiple kernels, each of which implements
-functionality for a specific *dispatch key*; for example, CPU, CUDA or Autograd.
+functionality for a specific *dispatch key*; for example, CPU, CUDA, AutogradCPU
+or AutogradCUDA.  Starting from 1.8 release the dispatcher allows separate registration
+to Autograd backend dispatch keys, and introduced the notion of alias dispatch keys
+for `dispatch table precomputation <https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/README.md#dispatch>`_.
 The dispatcher determines what the highest priority dispatch key is at the time
 you call an operator (this is done by looking at both the tensor arguments as
 well as some thread local state), and transfers control to the kernel for that
 dispatch key.  The end effect is that when you call an operator, we first
-execute the Autograd kernel, and then we redispatch to the CPU or CUDA kernel
-depending on the device types of the passed in tensors.
+check the device type of the passed in tensors and execute the corresponding Autograd
```

Comment:
I think this last sentence raises more questions than it answers for the careful reader:
- it says we redispatch only to CPU or CUDA - easily fixed
- it says we always execute an Autograd kernel - accurate for now, but confusing unless you're pretty familiar with PT inside baseball (why would we call this kernel during inference?). 
- it doesn't mention any of the other dispatch keys "above" autograd, which is going to be confusing if you *are* familiar with PT inside baseball

Rather than overstuff this first part of the tutorial, I'd suggest just cutting the last sentence. That leaves them with the "determines the highest priority dispatch key" mental picture, which is accurate and clear. But if we definitely want to mention Autograd here, I don't think we can get away without some clarification per above.